src/components/organisms/AdminVenueView/components/EventsView/EventsView.tsx

```diff
@@ -48,25 +56,50 @@ export const EventsView: React.FC<EventsViewProps> = ({ venueId, venue }) => {
 
   const renderedEvents = useMemo(
     () =>
-      events?.map((event) => {
-        return (
-          <TimingEvent
-            event={event}
+      events?.map((event) => (
+        <TimingEvent
+          event={event}
+          setShowCreateEventModal={setShowCreateEventModal}
+          setEditedEvent={setEditedEvent}
+          key={event.id}
+        />
+      )),
+    [events, setShowCreateEventModal, setEditedEvent]
+  );
+
+  const renderedSpaces = useMemo(() => {
+    const spaces = [...new Set(events?.map((event) => event.room))];
+    const setSpaceEvents = (space: string) =>
```

Comment:
this `setSpaceEvents` looks more like a getter function that returns `room === space` events
maybe it would be better to name it smth like `getSpaceRooms` or smth

Reply:
`getSpaceEvents` is more clear, like I'm getting the list of events for specific `space`

Fixed in 2afbf348cf9b4c04b752967251109f8b622ed0d6